api-report/tree.api.md

```diff
@@ -1185,6 +1186,16 @@ export interface PathRootPrefix {
     rootFieldOverride?: FieldKey;
 }
 
+// @alpha
+export interface PathVisitor {
+    // (undocumented)
+    onDelete(path: FieldUpPath, index: number, count: number): void;
```

Comment:
A field up path + index is just an UpPath (since it selects a location within the field), so this API choice seems a bit odd.

Reply:
Very true. Corrected along your suggestion.